doc/en/user/source/styling/sld/extensions/rendering-selection.rst

```diff
@@ -0,0 +1,305 @@
+.. _rendering_selection:
+
+Rendering Selection
+====================
+
+GeoServer provides two ``VendorOptions`` to define whether a particular element ``Rule``, ``FeatureTypeStyle`` or ``Symbolizer`` should be applied to a ``getLegendGraphic`` output or to a ``getMap`` output.
+The two options are named ``renderingMap`` and ``renderingLegend`` and can take a value of ``true`` or ``false`` eg.:
+
+* - <VendorOption name="renderingMap">false</VendorOption>
```

Comment:
To be updated based on the geotools PR changes.

Reply:
fixed